plotsearch/serializers.py

```diff
@@ -598,3 +601,42 @@ class Meta:
     def create(self, validated_data):
         validated_data["form"] = initialize_area_search_form()
         return super().create(validated_data)
+
+
+class InformationCheckSerializer(
+    EnumSupportSerializerMixin, serializers.ModelSerializer
+):
+    mark_all = serializers.BooleanField(write_only=True)
+    answer = serializers.IntegerField(source="entry_section.answer_id")
+    identifier = serializers.CharField(write_only=True)
+    type = serializers.CharField(write_only=True)
+
+    class Meta:
+        model = InformationCheck
+        fields = (
+            "name",
+            "answer",
+            "state",
+            "preparer",
+            "comment",
+            "identifier",
+            "type",
+            "mark_all",
+        )
+
+    def create(self, validated_data):
```

Comment:
Hmm... I guess this API being POSTable (and the use case shown in the test file later) implies the current flow:

1. An application is sent in. Sections have no InformationCheck objects attached to them yet.
2. The administrative UI fetches an application and its check list, which is still empty. It knows the master list of checks that an applicant _should_ have, and fills any missing ones with a default blank dropdown.
3. When any dropdown is first changed, the UI then creates the object for that piece  of information on the fly, or if it already existed, then it gets modified normally.

Sure, I guess that works.

Reply:
I guess we could also create InfromationChecks when posting answer. That could be more easier way to do it.

Reply:
If it isn't too much trouble, it could maybe work better. That way we always have all the needed InformationCheck entries already available and keep the backend in charge of the ground truth on what checks there should be in the first place.

Reply:
I will change the logic to that way